Sources/StreamVideoSwiftUI/CallView/ParticipantsSpotlightLayout.swift

```diff
@@ -47,22 +48,37 @@ public struct ParticipantsSpotlightLayout<Factory: ViewFactory>: View {
                 viewFactory: viewFactory,
                 participants: participants,
                 frame: participantsStripFrame,
-                call: call
+                call: call,
+                itemsOnScreen: itemsVisibleOnScreen,
+                showAllInfo: true
             )
         }
     }
     
     private var topParticipantFrame: CGRect {
+        /// Top 
         .init(
             origin: frame.origin,
-            size: CGSize(width: frame.size.width, height: frame.size.height - thumbnailSize)
+            size: CGSize(width: frame.size.width, height: frame.height - participantsStripFrame.height - innerItemSpace)
         )
     }
 
+    private var itemsVisibleOnScreen: CGFloat {
+        if UIDevice.current.userInterfaceIdiom == .pad {
```

Comment:
I see this in several places, maybe we have `isIpad` global method/variable?

Reply:
Certainly, i've made it an extension on UIDevice. WDYT?

Reply:
yep, that's good 👍